docs/Mirrors.md

```diff
@@ -41,7 +41,41 @@ You can create a public AlmaLinux mirror in 6 easy steps:
    your mirror to the `mirrors.d` directory.
    You can use the [official AlmaLinux repo file](https://github.com/AlmaLinux/mirrors/blob/master/mirrors.d/repo.almalinux.org.yml)
    as an example. Your mirror does not have to provide all the protocols
-   that our main mirror provides, but either HTTP or HTTPS is required.
+   that our main mirror provides, but either HTTP or HTTPS is required. Format of a mirror's config is described below. 
+   Also, you can validate your config use some JSON online validator using [that JSON schema](https://github.com/AlmaLinux/mirrors/blob/yaml_snippets/json_schemas/mirror_config.json) and converting your config to JSON.  
+    
+    ```YAML
+    ---
+    name: <domain name of a mirror, e.g. almalinux.mirror.link>
+    cloud_type: <azure|aws. It's required if a mirror is located in AWS/Azure cloud>
+    address:
+      http: <http URL to a mirror, e.g. http://almalinux.mirror.link/almalinux>
+      https: <https URL to a mirror, e.g. https://almalinux.mirror.link/almalinux>
+      rsync: <rsync URL to a mirror, e.g. rsync://almalinux.mirror.link/almalinux>
+      ftp: <ftp URL to a mirror, e.g. ftp://almalinux.mirror.link/almalinux>
+    update_frequency: <update frequency of a mirror, e.g. 1h>
+    sponsor: <Name of a mirror's sponsor/holder, e.g. Some company name>
+    sponsor_url: <URL of a mirror's sponsor/holder, e.g. http://some.company.name>
+    email: <e-mail of a mirror's sponsor/holder, e.g. admin@some.company.name>
+    geolocation:
+      continent: <name of a continent, e.g. North America; than field is not mandatory>
+      country: <two-letter name of a country, e.g. US>
+      state_province: <name of a province, e.g. New York>
+      city: <name of a city, e.g. New York>
+    private: <true|false. A mirror is available in some local network if the param is true>
+    monopoly: <true|false. The mirrors list contains only this private mirror for a suitable client if param is true>
+    # That list is required if a mirror is local or cloud.
```

Comment:
"The list will be required if a mirror is local or cloud" a little bit more English style